### ISSUE TITLE
Question about idr_baf 

### ISSUE THREAD

**Issue by mingoo426** [brentp/peddy#82]
Hi there! 
I'm a student who's new in bioinformatics/genetics/github and I apologize in advance if this is not the right place to ask questions.. (if there's a forum where I could ask about peddy, I'll relocate this to that forum)

But I wanted to ask about **idr_baf.**

**Question 1.** According to the documentation for the inter-decile range, it states: "Large values indicated likely sample contamination.". But isn't this already fulfilled from the het_call / median depth graph? What additional insight can we get from the idr_baf that we cannot get from the graph?

**Question 2.** About heterozygosity. "Large values indicated likely sample contamination.". When you say large value, is this relative to the other samples? Or is this relative to the heterozygosity of a typical genomic sample of humans? 

Thank you so much!

Best,
Min Ku

**Comment by brentp** [comment]
Hi Min,
1. this is just another measure. the heterozygosity and the IDR_BAF should be correlated, but sometimes only one is high

2. yes, relative to other samples. so often you can see outliers in your cohort.

(also check out [somalier](https://github.com/brentp/somalier) which is the successor to peddy).

**Comment by mingoo426** [comment]
thank you so much! :)

**Comment by mingoo426** [comment]
Hi Brent, would it be ok to ask just one more question?

I noticed that on the '**het / homo for X chromosome graph**', for most of my **male** samples, the ratios were not 0 (but close enough). I checked my vcf file and some of the **male samples actually had het calls** for some of the variants on the X chr. 

I thought why this is so (they are supposed to have 0 het calls theoretically, right?) 

- Technical reasons:
    - for some reason, the read that came from autosome somehow got aligned to X-chromosome?
    - Sequencing error? (mutation was introduced in the read at the hard-to-read region? or homopolymeric region such as CCCCCC?)  

- Biological
    - gene on X chromosome somehow got duplicated? (But would be unlikely for my samples, since it appears to be systematic) 

These are some of the reasons I came up with... am I thinking in the right direction?
